### PR TITLE
Improve COS image document for E2E test

### PR DESCRIPTION
**Pull request by bsdnet**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
/kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Update the GCI/COS document to today.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE 

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

### CODE REVIEW

File: cluster/gce/gci/README.md [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
-of container-vm, the new Container-VM Image is based on the open source
-ChromiumOS project, allowing us greater control over the build management,
+[Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs),
+(previously Google Container-VM image a.k.a GCI) is a container-optimized OS image for the Google Cloud Platform (GCP). It is
+primarily for running Google services on GCP. Container-Optimized OS is an open
+source OS based on
+the open source [ChromiumOS project](https://www.chromium.org/chromium-os), allowing us greater control over the build management,
 security compliance, and customizations for GCP.
 
+Container-Optimized OS is open [source](https://cos.googlesource.com), and is released on milestones. Example milestones are
```
Comment (MHBauer) [inline cluster/gce/gci/README.md:10]: I would suggest giving people a bigger target and making the link [open source]

File: cluster/gce/gci/README.md [hunk 2]
```diff
@@ -1,11 +1,42 @@
...
+81,  84, 85. Each milestone will experience three release channels -- dev, beta and stable to reach
```
Comment (MHBauer) [inline cluster/gce/gci/README.md:11]: Not sure what to take from the "Example milestones are ######" sentence.

File: cluster/gce/gci/README.md [hunk 3]
```diff
@@ -1,11 +1,42 @@
...
+## COS in End-to-End tests
+
+Container-Optimized OS images are used by kubernetes [End-to-End tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) and
+[Node End-to-End tests](https://github.com/kubernetes/community/tree/master/contributors/devel/sig-node). To see current
+active releases, please refer to COS's [Release
+Notes](https://cloud.google.com/container-optimized-os/docs/release-notes). When
+choose images for testing, the latest LTS images is suggested for running
+presubmit, postsubmit or periodics tests since the image is stable and has
+latest bug and security fixes. For new features which need new kernel features,
+the latest dev or beta or stable channel can be used. When multiple COS images
```
Comment (MHBauer) [inline cluster/gce/gci/README.md:27]: Do we have any examples of dev/beta usage where we actually need it? I see some usages of dev-83 and beta-81, which seems wrongheaded at this current time of 81 being lts.

File: cluster/gce/gci/README.md [hunk 4]
```diff
@@ -1,11 +1,42 @@
...
+primarily for running Google services on GCP. Container-Optimized OS is an open
+source OS based on
+the open source [ChromiumOS project](https://www.chromium.org/chromium-os), allowing us greater control over the build management,
 security compliance, and customizations for GCP.
 
+Container-Optimized OS is open [source](https://cos.googlesource.com), and is released on milestones. Example milestones are
+81,  84, 85. Each milestone will experience three release channels -- dev, beta and stable to reach
+stability. The promotion between those channels are about six weeks.
+Starting milestone 69, for
+every 4 milestones, the last milestone will be promoted into LTS image.
```
Comment (MHBauer) [inline cluster/gce/gci/README.md:14]: Is it promoted to lts before or after or at the same time as it becomes 'stable'?

**Review by MHBauer — COMMENTED** [review]
Probably a good swap, but I'm not sure I'd ever find this if I was looking for it.

File: cluster/gce/gci/README.md (hunk 1 above)
Comment (bsdnet) [reply cluster/gce/gci/README.md:10]: Done

File: cluster/gce/gci/README.md (hunk 2 above)
Comment (bsdnet) [reply cluster/gce/gci/README.md:11]: This shows what number 81,  85 mean.  Any good suggestion?

File: cluster/gce/gci/README.md (hunk 3 above)
Comment (bsdnet) [reply cluster/gce/gci/README.md:27]: I do not have context on beta-81/dev-83. However, if we need new features and continuous integration/testing, we need those images.

File: cluster/gce/gci/README.md (hunk 4 above)
Comment (bsdnet) [reply cluster/gce/gci/README.md:14]: Updated.

**Comment by bsdnet** [comment]
/assign @mml @karan @spiffxp 

Please let me know what needs to be done for this to be merged.

Thanks

**Pull request merged**